doc/source/serve/handle-guide.md

```diff
@@ -2,55 +2,59 @@
 
 # ServeHandle: Calling Deployments from Python
 
-Ray Serve enables you to query models both from HTTP and Python. This feature
-enables seamless [model composition](serve-model-composition-guide). You can
-get a `ServeHandle` corresponding to deployment, similar how you can
-reach a deployment through HTTP via a specific route. When you issue a request
-to a deployment through `ServeHandle`, the request is load balanced across
-available replicas in the same way an HTTP request is.
-
-To call a Ray Serve deployment from python, use {mod}`Deployment.get_handle <ray.serve.api.Deployment>`
-to get a handle to the deployment, then use
-{mod}`handle.remote <ray.serve.handle.RayServeHandle.remote>` to send requests
-to that deployment. These requests can pass ordinary args and kwargs that are
-passed directly to the method. This returns a Ray `ObjectRef` whose result
-can be waited for or retrieved using `ray.wait` or `ray.get`.
+ServeHandle can be used to invoke your Serve deployment and deployment graphs using Python API.
+
+This is particularly useful for two use cases:
+- Calling deployments dynamically within the deployment graph.
+- Iterating and testing your application in Python.
+
```

Comment:
Those not complete sentences so no need to start with capital case or end with a period. Suggest :

This is particularly useful for two use cases:

- when calling deployments dynamically within the deployment graph
- when iterating and testing your application in Python

**OR**
This is particularly useful for two use cases when:

- calling deployments dynamically within the deployment graph
- iterating and testing your application in Python